pkg/datapath/linux/node.go

```diff
@@ -472,8 +472,8 @@ func (n *linuxNodeHandler) enableSubnetIPsec(v4CIDR, v6CIDR []*net.IPNet) {
 		ipsecIPv6Wildcard := &net.IPNet{IP: net.ParseIP(wildcardIPv6), Mask: net.CIDRMask(0, 0)}
 
 		n.replaceNodeIPSecInRoute(cidr)
-		spi, err = ipsec.UpsertIPsecEndpoint(cidr, ipsecIPv6Wildcard, ipsec.IPSecDirIn)
-		upsertIPsecLog(err, "CNI In IPv6", cidr, ipsecIPv6Wildcard, spi)
+		//spi, err = ipsec.UpsertIPsecEndpoint(cidr, ipsecIPv6Wildcard, ipsec.IPSecDirIn)
+		//upsertIPsecLog(err, "CNI In IPv6", cidr, ipsecIPv6Wildcard, spi)
```

Comment:
Lets break this into two commits one to slit up replaceState into an out and in part and then a second one to drop the extra state. That way we can keep track of exactly what changed later. Also commit messages please so we can remember what this was for next month when reading git log.

Reply:
also just delete the lines assume that is just a wip thing though.